sktime/forecasting/model_selection/_tune.py

```diff
@@ -65,8 +71,8 @@ def update_predict(
         """Call update_predict on the forecaster with the best found
         parameters.
         """
-        self.check_is_fitted("update_predict")
-        return self.best_forecaster_.update_predict(
+
+        return self.best_forecaster_._update_predict(
```

Comment:
currently, there is no `_update_predict` in the base class, so thsi should be delegated to `update_predict`